doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -290,7 +295,27 @@ using two ``ml.p4d.24xlarge`` instances:
         }
     )
 
-    pt_estimator.fit("s3://bucket/path/to/training/data")
+**Using PyTorch DDP with the ``torchrun`` backend**
```

Comment:
in preview, the double back quote here didn't function correctly

Reply:
removed the back quotes. I guess double-layer styling doesn't work

Reply:
thanks!